CrashReporter/languages/CMakeLists.txt

```diff
@@ -4,10 +4,10 @@ find_package(Qt5LinguistTools)
 
 file(GLOB TS_FILES "*.ts")
 qt5_create_translation(QM_FILES ${SOURCES} ${TS_FILES})
-add_custom_target(CrashReporter_translations DEPENDS ${QM_FILES})
+add_custom_target(CrashReporter_translations ALL DEPENDS ${QM_FILES})
 
 if(APPLE AND UNIX)
 	install(FILES ${QM_FILES} DESTINATION Grabber.app/Contents/Resources/crashreporter)
 else()
-	install(FILES ${QM_FILES} DESTINATION ${CMAKE_INSTALL_PREFIX}/crashreporter)
+	install(FILES ${QM_FILES} DESTINATION ${CMAKE_INSTALL_PREFIX}/share/Grabber/crashreporter)
```

Comment:
If `${CMAKE_INSTALL_PREFIX}` is already assumed (from your comment) maybe it can be removed here for consistency?